src/gam/__init__.py

```diff
@@ -1495,7 +1499,14 @@ def printShowDelegates(users, csvFormat):
 
 
 def deleteDelegate(users):
-    delegate = normalizeEmailAddressOrUID(sys.argv[5], noUid=True)
+    convertAlias = False
+    i = 5
+    if sys.argv[i].lower().replace('_', '') == 'convertalias':
+        convertAlias = True
+        i += 1
+    delegate = normalizeEmailAddressOrUID(sys.argv[i], noUid=True)
+    if convertAlias:
+        delegate = gapi_directory_users.get_primary(delegate)
     i = 0
```

Comment:
shouldn't we only convert delegator to primary below also if convertAlias is set?